templates/rhdm71-authoring-ha.yaml

```diff
@@ -0,0 +1,1216 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    iconClass: icon-jboss
+    tags: rhdm,jboss,authoring,ha
+    version: "1.0"
+    openshift.io/display-name: Red Hat Decision Manager 7.1 authoring environment (HA, persistent, with https)
```

Comment:
In the image descriptor Decision Manager was renamed to Decision Central, maybe we should do the same here?
(comment valid for all the other templates)

Reply:
Yeah but the DM authoring env *is* Decision Central, so I don't think we have to change it here.